sources/niUI/src/VideoDecoderTheoraPlayer.cpp

```diff
@@ -207,61 +219,21 @@ class cVideoDecoderTheoraPlayer : public ni::cIUnknownImpl<ni::iVideoDecoder,ni:
       }
       this->SetTime(currentTime);
     }
-
     return eTrue;
   }
 
-  void _UpdateTarget() {
```

Comment:
where did this go? it would probably be better easier to follow if the changes were made here rather than inlining this function - makes the changes a lot easier to review